project/web/balancer.sls

```diff
@@ -209,17 +198,26 @@ link_key:
     - watch_in:
       - service: nginx
 
-# Once a week, renew our cert(s) if we need to. This will only renew them if
+# Twice a day, renew our cert(s) if we need to. This will only renew them if
 # they're within 30 days of expiring, so it's not a big burden on the certificate
-# service.  https://letsencrypt.readthedocs.org/en/latest/using.html#renewal
-renew_letsencrypt:
+# service.  (In fact, it's recommended to be this often, in the rare case of a
+# LetsEncrypt initiated revocation) https://certbot.eff.org/#ubuntutrusty-nginx
+renew_certbot:
   cron.present:
-    - name: env XDG_DATA_HOME=/root/letsencrypt {{ letsencrypt_dir }}/letsencrypt-auto renew; /etc/init.d/nginx reload
-    - identifier: renew_letsencrypt
+    - name: certbot-auto renew --quiet --no-self-upgrade && /etc/init.d/nginx reload
```

Comment:
Is it documented that `certbot renew` only exits successfully if it's changed a certificate? If not, we might prefer adding `--post-hook "service nginx reload"` to its options rather than using `&&` to trigger nginx to reload certificates.